core/storage/classifier/gae_models.py

```diff
@@ -111,3 +116,87 @@ def create(
 
         classifier_model_instance.put()
         return instance_id
+
+
+class TrainClassifierJobModel(base_models.BaseModel):
+    """Model for storing classifier training jobs.
+
+    The id of instances of this class has the form
+    {{exp_id}}.{{random_hash_of_16_chars}}
+    """
+
+    # The ID of the algorithm used to create the model.
+    algorithm_id = ndb.StringProperty(required=True, choices=ALGORITHM_CHOICES)
```

Comment:
Do you need indexed=True for this and/or any of the fields below? Seems to me like you might want it for everything but training_data...